builders/toolkit/ethereum-api/precompiles/cross-chain-transfers.md

```diff
@@ -0,0 +1,229 @@
+    Also, if the token being transferred is native to your appchain, the destination chain must have registered the foreign asset.
+
+### Remix Set Up {: #remix-set-up }
+
+You can interact with the XCM interface precompile using [Remix](https://remix.ethereum.org){target=\_blank}. To add the precompile to Remix, you will need to:
+
+1. Get a copy of [`XcmInterface.sol`](https://github.com/moondance-labs/tanssi/blob/master/test/contracts/solidity/XcmInterface.sol){target=\_blank}
+2. Paste the file contents into a Remix file named `XcmInterface.sol`
+
+### Compile the Contract {: #compile-the-contract }
+
+Next, you will need to compile the interface in Remix:
+
+1. Click on the **Compile** tab, second from top
+2. Compile the interface by clicking on **Compile XcmInterface.sol**
+
+![Compiling XcmInterface.sol](/images/builders/toolkit/ethereum-api/precompiles/xcm-interface/xcm-interface-1.webp)
+
+When the compilation is completed, you will see a green checkmark next to the **Compile** tab.
+
+### Access the Contract {: #access-the-contract }
+
+Instead of deploying the precompile, you will access the interface given the address of the precompiled contract:
+
+1. Click on the **Deploy and Run** tab directly below the **Compile** tab in Remix. Please note that the precompiled contracts are already accessible at their respective addresses. Therefore, there is no deployment step
+2. Make sure **Injected Provider - Metamask** is selected in the **ENVIRONMENT** dropdown. Once you select **Injected Provider - Metamask**, you may be prompted by MetaMask to connect your account to Remix if it's not already connected
+3. Make sure the correct account is displayed under **ACCOUNT**
+4. Ensure **XCM - XcmInterface.sol** is selected in the **CONTRACT** dropdown. Given that it is a precompiled contract, there is no deployment step. Instead, you are going to provide the address of the precompile in the **At Address** field
+5. Provide the address of the precompile: `{{networks.dancebox.precompiles.xcmInterface}}` and click **At Address**
+
+![Access the address](/images/builders/toolkit/ethereum-api/precompiles/xcm-interface/xcm-interface-2.webp)
+
+The **XCM Interface** precompile will appear in the list of **Deployed Contracts**.
+
+### Send Tokens Over to Another EVM-compatible Appchain {: #transfer-to-evm-chains }
+
+To send tokens over to an account in another EVM-compatible appchain, please follow these steps:
+
+1. Expand the **transferAssetsToPara20** function
+2. Enter the appchain ID (paraId)
+3. Enter the ECDSA destination account (beneficiary)
+4. Specify the tokens to be transferred. Note that this parameter is an array that contains at least one asset. Each asset is specified by its address and the total amount to transfer
+
+--8<-- 'text/builders/toolkit/ethereum-api/precompiles/xcm-interface/erc-20-note.md'
```

Comment:
Here is where we would indent the snippet

Reply:
fixed the snippet and indented it in the markdown where it is used to align as needed